AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -132,6 +132,12 @@
     'Kajiya': 2,
 }
 
+# Physical Sky Intensity Mode
+PHYSICAL_SKY_INTENSITY_MODE = {
+    'Ev100': 4,
+    'Nit': 3,
+}
+
```

Comment:
I think this was already merged as part of physical sky component. your branch might be out of date.

Reply:
Fetch, pull and merge worked! fixed!